scripts/netContain/ReleaseContainer.sh

```diff
@@ -1,83 +0,0 @@
-contiv_version=""
```

Comment:
This script is just moved over to `release_image.sh` and formatted. I know marketing material refers to Docker images as containers, but a container is instantiated from an image, similar to how an object is an instance of a class in some programming languages.